tests/pii_replacer/test_config.py

```
@@ -0,0 +1,310 @@
+            ),
+        ],
+        standalone_columns_to_replace=[
+            PiiColumnPlan(column_name="notes", entity_type=PiiEntity.free_text),
+        ],
+    )
+    assert _plan_column_counts(plan) == (1, 2, 1)
+
+
+def test_replacement_plan_source():
+    assert _replacement_plan_source(ReplacePiiConfig()) == "auto_discovery"
+    assert _replacement_plan_source(ReplacePiiConfig(replacement_plan="/tmp/plan.json")) == "/tmp/plan.json"
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/plan.json"

(S108)